src/main.rs

```diff
@@ -124,6 +124,7 @@ fn main() {
                     Core::CortexM4 => "M4",
                     Core::CortexM3 => "M3",
                     Core::CortexM33 => "M33",
+                    Core::CortexM7 => "M7",
```

Comment:
This enum variant is still missing in probe-rs so I will wait with the merge until probe-rs has the feature in :)

Reply:
Let's merge this!